src/cgroups/manager.rs

```diff
@@ -40,6 +41,7 @@ impl Manager {
                 "hugetlb" => Hugetlb::apply(linux_resources, &subsys.1, pid)?,
                 "memory" => Memory::apply(linux_resources, &subsys.1, pid)?,
                 "pids" => Pids::apply(linux_resources, &subsys.1, pid)?,
+                "cpu" => Cpu::apply(linux_resources, &subsys.1, pid)?,
```

Comment:
You imported the enum used for dispatch, but you didn't actually import your controller. So this is not actually declared in scope.